WORKSPACE

```diff
@@ -1,10 +1,9 @@
-'''
+"""
```

Comment:
Linter says:
Only the first line of the file could be docstring using `"""...."""` else all other comment lines should use `#`. Using `'''.....'''` is not recommended. 

Lint failure we get without this fix
```
WORKSPACE # reformat
```

Reply:
Makes sense--thanks for fixing this!